### PR TITLE
Tracking Item and Misc. Google Analytics Updates

### PR DESCRIPTION
**Pull request by utku-ozturk**
This PR fixes the bugs and styling issues listed below:

1. The submissions/usage tabs' background color is green instead of blue
Screen Shot 2024-02-20 at 14 58 33

2. (High Priority) Since a bug prevents collecting submission center and file type dimensions in file views, data aggregation is shown as `(not set)`
Screen Shot 2024-02-20 at 15 07 12

3. (High Priority) Clickable file tooltips are incorrect in file downloads chart (also check: https://github.com/smaht-dac/foursight-smaht/pull/9)
Screen Shot 2024-02-14 at 16 31 10

4. tracking_item schema still has 4DN-specific labels and titles:
Screen Shot 2024-02-20 at 15 43 13

5. Various unused reports and aggregations removed in statistics page
6. `tracking_item` py test added

### CODE REVIEW

File: src/encoded/static/components/app.js [hunk 1]
```diff
@@ -209,11 +209,40 @@ export default class App extends React.PureComponent {
...
+        
+                const categories = Array.isArray(itemType) ? itemType.slice().reverse().slice(1) : [];
+                const prodItem = {
+                    'item_id': itemID || itemUUID,
+                    'item_name': display_title || title || null,
+                    'item_category': categories.length >= 1 ? categories[0] : "Unknown",
+                    'item_category2': categories.length >= 2 ? categories[1] : "Unknown",
+                    'item_category3': categories.length >= 3 ? categories[2] : "Unknown",
+                    'item_brand': submissionCenterTitle || null,
+                    'experiment_type': assayTitle || null,
```
Comment (Bianca-Morris) [inline src/encoded/static/components/app.js:233]: Out of curiosity, why is this still `experiment_type`?

File: src/encoded/static/components/app.js [hunk 2]
```diff
@@ -209,11 +209,40 @@ export default class App extends React.PureComponent {
...
+                    'item_variant': file_type || null
+                };
+
+                return prodItem;
+            }
+        }, analyticsConfigurationOptions);
+
         // ANALYTICS INITIALIZATION; Sets userID (if any), performs initial pageview track
         // TODO: add a google analytics configuration for smaht; currently this will never run
```
Comment (Bianca-Morris) [inline src/encoded/static/components/app.js:242]: Would be good to remove this comment, since this is no longer true.

**Review by Bianca-Morris — APPROVED** [review]
Haven't tested this locally, but code looks good.

File: src/encoded/static/components/app.js (hunk 1 above)
Comment (utku-ozturk) [reply src/encoded/static/components/app.js:233]: @Bianca-Morris  we are still using analytics utils in shared-portal-components that was previously implemented for 4DN.

**Pull request merged**